pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -106,7 +105,7 @@ func NewConfigObserver(
 				[]string{"apiServerArguments", "cloud-config"}),
 			featuregates.NewObserveFeatureFlagsFunc(
 				nil,
-				sets.NewString("IPv6DualStack"), // IPv6DualStack is bugged, don't turn it on
+				nil,
```

Comment:
cc @deads2k ... we removed that feature gate, so we don't have to black list this anymore...